### ISSUE TITLE
Nothing is rendered

### ISSUE THREAD

**Issue by anchetaWern** [FaridSafi/react-native-gifted-chat#2007]
#### Issue Description

I'm getting a blank screen when running the example.

#### Steps to Reproduce / Code Snippets

Slightly modify the example here: https://github.com/FaridSafi/react-native-gifted-chat#example
and use it in the default React Native blank template.

```
// App.js
import React from 'react';
import {SafeAreaView, StyleSheet, ScrollView, StatusBar} from 'react-native';

import {Colors} from 'react-native/Libraries/NewAppScreen';

import Chat from './src/Chat';

const App: () => React$Node = () => {
  return (
    <>
      <StatusBar barStyle="dark-content" />
      <SafeAreaView>
        <ScrollView
          contentContainerStyle={{
            flexDirection: 'row',
          }}
          contentInsetAdjustmentBehavior="automatic"
          style={styles.scrollView}>
          <Chat />
        </ScrollView>
      </SafeAreaView>
    </>
  );
};

const styles = StyleSheet.create({
  scrollView: {
    backgroundColor: Colors.lighter,
  },
});

export default App;
```

```
// src/Chat.js
import React, {useState, useCallback, useEffect} from 'react';
import {GiftedChat} from 'react-native-gifted-chat';

const Chat = () => {
	const [messages, setMessages] = useState([]);

	useEffect(() => {
		setMessages([
			{
				_id: 1,
				text: 'Hello developer',
				createdAt: new Date(),
				user: {
					_id: 1,
					name: 'React Native',
					avatar: 'https://placeimg.com/140/140/any',
				},
			},
			{
				_id: 2,
				text: 'Hellos developer',
				createdAt: new Date(),
				user: {
					_id: 1,
					name: 'dd',
					avatar: 'https://placeimg.com/140/140/any',
				},
			},
		]);
	}, []);

	const onSend = useCallback((messages = []) => {
		setMessages((previousMessages) =>
			GiftedChat.append(previousMessages, messages),
		);
	}, []);

	return (
		<GiftedChat
			messages={messages}
			onSend={(messages) => onSend(messages)}
			user={{
				_id: 1,
			}}
		/>
	);
};

export default Chat;
```

#### Expected Results

Chat UI is rendered.

#### Additional Information

* Nodejs version: 14.4.0
* React version: 16.13.1
* React Native version: 0.63.4
* react-native-gifted-chat version: 0.16.3
* Platform(s) (iOS, Android, or both?): only tested on iOS
* TypeScript version: N/A

**Comment by oxbits** [comment]
in my app it renders on my iPhone 6s 14.4

on my friend's iPhone XR it is blank 

so i built a vanilla gifted chat app and that appears on both devices

**Comment by anchetaWern** [comment]
@oxbits what do you mean by vanilla gifted chat app?

**Comment by mariomurrent-softwaresolutions** [comment]
@anchetaWern If you render something different than then chat, e.g an Text does it appear? 
So make sure it is not any issue with the container ...

**Comment by anchetaWern** [comment]
@mariomurrent-softwaresolutions yeah. I did try that. View and Text in the same container I'm rendering gifted chat, it worked. For some reason only gifted chat doesn't show

**Comment by mariomurrent-softwaresolutions** [comment]
Then it seems that is is an layout issue? Can you maybe provide an online working example?

**Comment by basithrehman** [comment]
Gifted chat working fine on my iphone 12 but its not appearing in few iphone6 and iphone 6+. Anyway to solve this issue?  @FaridSafi

**Comment by basithrehman** [comment]
@oxbits How did you create vanilla gifted chat?

**Comment by oxbits** [comment]
my issue was something else

for the vanilla gifted chat i just created a new blank react native app
and replace the hello world at App.js with gifted-chat example code from the readme
this proved it worked on both devices

then i realized it had nothing to do with gifted chat
it was unexpected data from an API call that the react native client code was choking on and crashing

sorry if this is not more helpful

**Comment by akrv** [comment]
Having the same issue on the iOS simulator with iPhone 12. 
Is there any fix or workaround?


expo: 40.0.1
react-native-gifted-chat version: 0.16.3

**Comment by mariomurrent-softwaresolutions** [comment]
> Having the same issue on the iOS simulator with iPhone 12.
> Is there any fix or workaround?
> 
> expo: 40.0.1
> react-native-gifted-chat version: 0.16.3

Can you maybe post a working snack or something similar? Otherwise it is hard to help you.
It works fine for me on the iPhone 12 simulator as well as on the device.

**Comment by akrv** [comment]
Hi @mariomurrent-softwaresolutions 
thank you for your response. 
I solved the issue. The component works fine, the problem was on my side.
I am going to try and explain what I did and what I had.

I had UI-Kitten top navigation bar inside a scroll view and a view component.
GiftedChat manages to scroll, so I removed that.

then, I added flex = 1 in the styles of the view.
`    ChatContainer: {

        flex: 1
    },`

The minimal working example functions now.

As related to the problem: I can confirm iOS simulator iPhone 12 and hardware iPhone 11 Pro works fine.
